ui-super-heroes/src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+quarkus.quinoa.package-manager-install=true
+quarkus.quinoa.package-manager-install.node-version=16.16.0
```

Comment:
I've just remembered, looking at the broken build, why we're on node 16: https://stackoverflow.com/questions/69692842/error-message-error0308010cdigital-envelope-routinesunsupported

It seems like various bits of the ecosystem need to catch up with node 18. Maybe, when we do #290 we can move up node versions, but for now, I think we should stay on 16.

Reply:
That's interesting because its currently running on node 18...

https://github.com/quarkusio/quarkus-super-heroes/blob/2f99db07cfe69aa4313a9337fb2c76761d66850f/ui-super-heroes/Dockerfile#L23

Reply:
Ah, it's this line, a little bit further down in the same dockerfile, that makes it work 

```
# NODE_OPTIONS=--openssl-legacy-provider is a hack, but until the UI has been completely re-written, this will have to do
ENV HTTP_PORT=8080 \
    NODE_ENV=production \
    NODE_OPTIONS=--openssl-legacy-provider
```

So we could do the same setting of `NODE_OPTIONS`, but the effect is similar to downgrading to 16; in either case we're running with the less-secure SSL, so I'm not sure there's much in it.

Reply:
Yeah I guess it doesn't matter. We should probably put that in the node options anyways just to keep it the same.

Reply:
My worry with putting it in the node options is that then we have to remember to take it out of the node options when we do #290 ... (We also have to remember to update Node when we do #290, but that seems maybe easier to remember?)

Reply:
I guess you're right